### PR TITLE
Review appveyor build

### PR DESCRIPTION
**Pull request by AnthonySteele**
Review appveyor build scripts and data files
Make it simpler and more like how https://github.com/justeat/JustSaying/ does it
lets see if it works!

### CODE REVIEW

**Comment by AnthonySteele** [comment]
There's a package output from it now: https://ci.appveyor.com/project/justeattech/nlog-structuredlogging-json/build/2.0.0-beta-build219-n8ctqgxj/artifacts

Same size (55Kb)  and same files inside as the last build on master, so I don't think anything got lost.

File: version.props [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<Project>
+  <PropertyGroup>
+    <VersionPrefix>2.0.0</VersionPrefix>
+    <VersionSuffix>beta</VersionSuffix>
```
Comment (slang25) [inline version.props]: `beta1` maybe? or do you intend to go with just `beta`?

**Review by slang25 — APPROVED** [review]
Looks good, just one question/check but not a blocker

File: version.props (hunk 1 above)
Comment (AnthonySteele) [reply version.props]: Good spot, I will change that. Until now we just used the build counter as the beta number, e.g. https://www.nuget.org/packages/NLog.StructuredLogging.Json/2.0.0-beta-180

I'm hoping that the next build on master is an `RC` not a `beta` anyway.

**Comment by slang25** [comment]
Yeah, so this is a conscious change that was made for JustSaying, I intend to hand version all releases, so nuget should look like:
- 5.0.0-alpha1
- 5.0.0-alpha2
- 5.0.0-beta1
- 5.0.0-rc1
- 5.0.0
- 5.0.1
- 5.0.2
- 5.1.0
- 6.0.0-alpha1

Rather than the big numbers we used to use, it's cleaner for consumers that way, CI builds will have build numbers in the package name. The build number will still be in the assembly file version as the 4th part.

**Comment by slang25** [comment]
👍 Looks good

**Pull request merged**